source/frontend/src/features/mapSideBar/disposition/form/DispositionPropertiesSubForm.tsx

```diff
@@ -33,12 +89,23 @@ const DispositionPropertiesSubForm: React.FunctionComponent<DispositionPropertie
       );
       const bounds = geoJSON(locations).getBounds();
 
-      mapMachine.requestFlyToBounds(bounds);
+      requestFlyToBounds(bounds);
     }
   };
 
+  const handleAddToSelection = useCallback(() => {
+    prepareForCreation([selectedFeatureDataset]);
+  }, [prepareForCreation, selectedFeatureDataset]);
+
+  useEffect(() => {
+    if (exists(localRef.current) && propertyForms.length > 0) {
+      addPropertiesToCurrentFile(localRef, 'fileProperties', propertyForms, noop);
```

Comment:
I think the problem here might stem from the fact that "addPropertiesToCurrentFile" function is expecting a formikRef but this component receives the FormikProps directly so perhaps we would need another "addProperties" function that takes formikprops instead of a ref?